pkg/utils/metrics/metrics.go

```diff
@@ -0,0 +1,58 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I was wondering that too, but it also might be my personal twitch I get for anything in a `utils` directory

Reply:
I am planning on adding more metrics soon. Other metrics are a bit harder to integrate because the code is not organised with instrumentation in mind so I'll be doing that incrementally in a series of PRs. 

I prefer the all-metrics-in-a-single-place approach as it is easier to audit and helps to implicitly enforce conventions on new metrics -- which is harder to do when they are strewn all over the place.